source/beamline/AMPVControl.cpp

```diff
@@ -50,6 +56,17 @@ AMReadOnlyPVControl::AMReadOnlyPVControl(const QString& name, const QString& rea
 
 }
 
+
+void AMReadOnlyPVControl::enableLimitMonitoring()
```

Comment:
Unfortunately, @helfrij, since limits are things you need to trust are implemented at the EPICS layer it is actually the reverse case where you'd only enable limit monitoring for controls you know have limits.